infra/ansible/Dockerfile

```diff
@@ -20,8 +20,7 @@ ARG ansible_vars
 RUN ansible-playbook -vvv playbook.yaml -e "stage=release" -e "${ansible_vars}" --tags "install_deps"
 
 WORKDIR /tmp/wheels
-COPY --from=build /src/pytorch/dist/*.whl ./
-COPY --from=build /src/pytorch/xla/dist/*.whl ./
+COPY --from=build /dist/*.whl ./
```

Comment:
does it mean we don't install the torch wheel and torch_xla wheel?

Reply:
All whls should be installed. I found this from the e2e test dockerfile

Reply:
Interesting

Reply:
@vanbasten23 FYI it's from https://github.com/pytorch/xla/blob/master/infra/ansible/e2e_tests.Dockerfile#L27-L32